<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Programs\wpf-diet-recorder\DietRecorder_Tests\DietRecorder_UnitTests.nunit" total="6" errors="0" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2010-04-30" time="22:37:23">
  <environment nunit-version="2.5.4.10098" clr-version="2.0.50727.4927" os-version="Microsoft Windows NT 6.1.7600.0" platform="Win32NT" cwd="C:\Program Files\NUnit 2.5.4\bin\net-2.0" machine-name="ITT04506" user="gcrofto" user-domain="NETWORKS" />
  <culture-info current-culture="en-GB" current-uiculture="en-US" />
  <test-suite type="Project" name="C:\Programs\wpf-diet-recorder\DietRecorder_Tests\DietRecorder_UnitTests.nunit" executed="True" result="Success" success="True" time="0.648" asserts="0">
    <results>
      <test-suite type="Assembly" name="C:\Programs\wpf-diet-recorder\DietRecorder_Tests\bin\Debug\DietRecorder_Tests.dll" executed="True" result="Success" success="True" time="0.639" asserts="0">
        <results>
          <test-suite type="Namespace" name="DietRecorder_Tests" executed="True" result="Success" success="True" time="0.637" asserts="0">
            <results>
              <test-suite type="Namespace" name="DataAccess" executed="True" result="Success" success="True" time="0.616" asserts="0">
                <results>
                  <test-suite type="TestFixture" name="Repository_Tests" executed="True" result="Success" success="True" time="0.613" asserts="0">
                    <results>
                      <test-case name="DietRecorder_Tests.DataAccess.Repository_Tests.LoadUserList_NoUsersInDatabase_ReturnsEmptyList" executed="True" result="Success" success="True" time="0.574" asserts="1" />
                      <test-case name="DietRecorder_Tests.DataAccess.Repository_Tests.LoadUserList_UsersInDatabase_ReturnsUsers" executed="True" result="Success" success="True" time="0.022" asserts="1" />
                    </results>
                  </test-suite>
                </results>
              </test-suite>
              <test-suite type="Namespace" name="Model" executed="True" result="Success" success="True" time="0.015" asserts="0">
                <results>
                  <test-suite type="TestFixture" name="User_Tests" executed="True" result="Success" success="True" time="0.012" asserts="0">
                    <results>
                      <test-case name="DietRecorder_Tests.Model.User_Tests.GetValidationFailures_NameHas40Characters_NoMessageReturned" executed="True" result="Success" success="True" time="0.003" asserts="1" />
                      <test-case name="DietRecorder_Tests.Model.User_Tests.GetValidationFailures_NameHas41Characters_ReturnsCorrectMessage" executed="True" result="Success" success="True" time="0.002" asserts="2" />
                      <test-case name="DietRecorder_Tests.Model.User_Tests.GetValidationFailures_NameHasOneCharacter_NoMessageReturned" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                      <test-case name="DietRecorder_Tests.Model.User_Tests.GetValidationFailures_NameHasZeroCharacters_ReturnsCorrectMessage" executed="True" result="Success" success="True" time="0.000" asserts="2" />
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>